### PR TITLE
added sivasai's intro

### PR DESCRIPTION
**Pull request by sivasaisilla**
Hey I have added my intro in readme file, hope my pr will be merged

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by sivasaisilla** [comment]
My intro is not showing in the readme file!! @Panquesito7

**Comment by Panquesito7** [comment]
> My intro is not showing in the readme file!! @Panquesito7

Your changes will be reflected once the pull request is approved by two or more maintainers and merged. 🙂

File: README.md
Comment (KendallDoesCoding) [inline README.md]: I don't really like that you state that your a noob in opensource, that's not the right words and it makes you sound bad. Instead, let's put New To Open-Source :)!, what do you say?

Comment (sivasaisilla) [reply README.md]: Sure @KendallDoesCoding

**Pull request merged**
